config/core/rootfs-configs.yaml

```diff
@@ -143,6 +143,7 @@ rootfs_configs:
       - ca-certificates
       - iproute2
       - jdim
+      - libasound2
```

Comment:
Building a set of rootfs images on staging with this included:
https://bot.staging.kernelci.org/job/rootfs-build-trigger/115/

Reply:
They will be used on staging with this temporary PR:
https://github.com/kernelci/kernelci-core/pull/1128